external-crates/move/crates/move-compiler/src/typing/core.rs

```diff
@@ -1743,9 +1743,10 @@ pub fn public_testing_visibility(
         return None;
     }
 
-    // TODO support sui init functions
-    // let flavor = env.package_config(package).flavor;
-    // flavor == Flavor::Sui && callee_name.value() == INIT_FUNCTION_NAME
+    #[cfg(debug_assertions)]
+    if _callee_name.value() == crate::sui_mode::INIT_FUNCTION_NAME {
```

Comment:
```suggestion
    if callee_name.value() == crate::sui_mode::INIT_FUNCTION_NAME {
```
Would need to un-underscore the variable